test/endpoints/agentDownload.test.ts

```diff
@@ -1,13 +1,66 @@
 import worker from '../../src/index'
-import { WorkerEnv } from '../../src/env'
+import { Defaults, WorkerEnv } from '../../src/env'
 import { config } from '../../src/config'
 
 const workerEnv: WorkerEnv = {
+  ...Defaults,
```

Comment:
Let's avoid using `Defaults` object in the tests because this object is not related to tests. We can either put `FPJS_CDN_URL=null` and `FPJS_INGRESS_BASE_HOST=null` everywhere, or we can put question marks in front of these fields in the `WorkerEnv` type, like `FPJS_CDN_URL?: string | null`, whichever works for you @TheUnderScorer. This applies to all test files.